data/virt_autotest/kubevirt_tests/core-tests.conf

```diff
@@ -0,0 +1,31 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Kubevirt core test configuration
+# Maintainer: Nan Zhang <nan.zhang@suse.com>
```

Comment:
Would you please also add our team ML to maintainer, qe-virt@suse.de?

Reply:
Yes, updated.